build.sbt

```diff
@@ -86,5 +86,3 @@ publishTo := {
 }
 
 credentials += Credentials(Path.userHome / ".sonatype" / "credentials.ini")
-
-useGpg := true
```

Comment:
This is required to publish artifact to Maven Central. If it breaks built at your machine, just install the plugin: https://www.scala-sbt.org/sbt-pgp/usage.html